Automattic/jetpack
# Stats: Move Jetpack Stats Widget in WP Dashboard to Own Class

#### Changes proposed in this Pull Request:

* This PR is part of a larger refactor of our stats module, starting by moving the Jetpack stats widget in the WordPress dashboard into its own class, beginning with removing the widget code from class.jetpack.php.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Switch to this branch, preferably on a site that has some stat data.
* Make sure the stats widget on the WordPress Dashboard (not the Jetpack one), continues to work.
* Check for any regressions
